test/integration/patients/worklist/worklist.js

```diff
@@ -3127,7 +3127,7 @@ context('worklist page', function() {
         fx.data[0].attributes.updated_at = `${ currentYear }-01-05`;
         fx.data[0].relationships.patient.data.id = '1';
         fx.data[0].relationships.owner.data = {
-          id: '55555',
+          id: '11111',
```

Comment:
Use the existing `Clinician McTester` clinician fixture instead of `Test Clinician` that we create for only this test.